core/modules/query/TableRef.cc

```diff
@@ -225,7 +230,11 @@ void TableRef::putTemplate(QueryTemplate& qt) const {
     if (QueryTemplate::DEFINE == aliasMode) {
         if (hasAlias()) {
             qt.append("AS");
-            qt.append("`" + _alias + "`");
+            if (quoteIdentifiers) {
+                qt.append("`" + _alias + "`");
+            } else {
+                qt.append(_alias);
+            }
```

Comment:
If you accepted my suggestion to encapsulate the quoting into an inline method `sqlQuotedId` then these 5 lines could be replaced with just 1:
```
qt.apend(quoteIdentifiers ? sqlQuotedId(_alias) : _alias);
```
I'm not insisting on this. Just a suggestion.

Reply:
thanks for noticing this repeated pattern. You made me realize the whole quoting thing could go in QueryTemplate, and I can add a function called `QueryTemplate::appendIdentifier(string)` that does the Right Thing. I'll upload changes soon.